src/test/java/uk/gov/ons/ctp/integration/ratelimiterclient/RateLimiterClientIntegrationTest.java

```diff
@@ -120,15 +124,18 @@ private void invokeJsonEndpoint(String narrative, HttpStatus expectedHttpStatus)
 
     HttpStatus actualHttpStatus;
     try {
+      String ipAddress = useIP ? "1.23.34.45" : null;
+      String telNo = useTelNo ? "0123 3434333" : null;
+
       // Get client to call /json endpoint
       RateLimitResponse response =
           client.checkRateLimit(
               RateLimiterClient.Domain.RH,
               product,
               CaseType.HH,
-              "1.23.34.45",
+              ipAddress,
               new UniquePropertyReferenceNumber("24234234"),
-              "0123 3434333");
+              telNo);
       System.out.println("Response:");
```

Comment:
would prefer not to have any println output in the tests. Goes for all the tests.

Reply:
I would usually agree, but in this case the test is more of a manual 'integration' test, and really exists for pre-cucumber manual testing. 
The  invokeJsonEndpoint() method is not called unless an environment variable has been set to point at a rateLimiter service.

Reply:
we agreed a rename to "_IT" convention would be sufficient to take it out of the maven test run.